flink-table/flink-table-planner/pom.xml

```diff
@@ -300,7 +300,9 @@ under the License.
 							<include>org.apache.calcite.avatica:*</include>
 
 							<!-- Calcite's dependencies -->
-							<include>com.esri.geometry:esri-geometry-api</include>
+							<include>org.locationtech.proj4j:proj4j</include>
```

Comment:
Can we exclude those dependencies? Which classes actually need them?

Reply:
Thanks for the pointing to that.
Yep, it seems `org.locationtech.proj4j:proj4j` and `org.locationtech.jts.io:jts-io-common` could be omitted.
`org.locationtech.jts:jts-core`  is used in `RexBuilder`, `RexLiteral` and other classes like 
https://github.com/apache/calcite/blob/413eded693a9087402cc1a6eefeca7a29445d337/core/src/main/java/org/apache/calcite/rex/RexBuilder.java#L62
so it's required